benchmark/sparse_blas/operations.cpp

```diff
@@ -566,6 +569,98 @@ class LookupOperation : public BenchmarkOperation {
 };
 
 
+bool validate_factorization(const Mtx* input, const Mtx* factors)
```

Comment:
perhaps 
```suggestion
bool validate_symbolic_factorization(const Mtx* input, const Mtx* factors)
```
because I was confused for a while wondering why there is no comparison between values.